packages/server/sources/fetch/src/types.ts

```
@@ -0,0 +1,36 @@
+import type { WalkerOS, Source as CoreSource } from '@walkeros/core';
+import type { SettingsSchema, CorsOptionsSchema } from './schemas';
+import { z } from '@walkeros/core/dev';
+
+export type Settings = z.infer<typeof SettingsSchema>;
+export type CorsOptions = z.infer<typeof CorsOptionsSchema>;
+export type InitSettings = Partial<Settings>;
+
+export interface Mapping {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)